tools/bigquery-zos-mainframe-connector/USING_WITH_BQ.md

```diff
@@ -0,0 +1,227 @@
+# Using with BigQuery
+
```

Comment:
add notes about the job logging table and what information is recorded

Reply:
Added to the main readme since the table is supported by some bq and gsutil commands.